src/gpu_buffer_manager.cpp

```diff
@@ -423,36 +430,45 @@ GPUBufferManager::createTableAndColumnInGPU(Catalog& catalog, ClientContext& con
 			auto &pk = constraint->Cast<UniqueConstraint>();
 			if (pk.HasIndex()) {
                 SIRIUS_LOG_DEBUG("Unique constraint on index {}", pk.GetIndex().index);
-                for (auto &col : pk.GetColumnNames()) {
-                    SIRIUS_LOG_DEBUG("Unique constraint on column {}", col);
-                }
                 unique_columns.push_back(pk.GetIndex().index);
-			} else {
-                for (auto &col : pk.GetColumnNames()) {
-                    SIRIUS_LOG_DEBUG("Unique constraint on column {}", col);
-                }
-            }
+			}
 		}
 	}
-    
-    //finding column_name in column_names
-    //convert column_name to uppercase
+
+    // Normalize names
+    string up_table_name = table_name;
```

Comment:
is there a reason why this case handling isn't done before it gets to the Sirius part of the code? Duckdb treats these as all case insensitive  can we rely on duckdb handling this?